h5py/h5d.pyx

```diff
@@ -395,3 +395,43 @@ cdef class DatasetID(ObjectID):
             """
             H5Drefresh(self.id)
 
+
+    IF HDF5_VERSION >= (1, 8, 11):
+
+        def write_direct_chunk(self, offsets, data, filter_mask=0, PropID dxpl=None):
```

Comment:
Please use the Cython types where appropriate (for e.g. data, filter_mask).  This will save some work in the body of the function; for example, you won't have to cast to `filters`.